src/Common/AsynchronousMetrics.cpp

```diff
@@ -90,6 +92,8 @@ AsynchronousMetrics::AsynchronousMetrics(
         openFileIfExists("/sys/fs/cgroup/cpu/cpu.cfs_period_us", cgroupcpu_cfs_period);
         openFileIfExists("/sys/fs/cgroup/cpu/cpu.cfs_quota_us", cgroupcpu_cfs_quota);
     }
+    if (!cgroupcpu_stat)
```

Comment:
(`cpuacct` is a v1 metric (and rightfully in the v1 section, l. 85). Perhaps it is time to stop supporting v1 at some point, it is obsolete since a decade now :-/ )